packages/react-lowcode/src/codegen/generation/generators/detail/mui-detail-generator.ts

```diff
@@ -140,6 +140,58 @@ export default class MuiDetailGenerator implements DetailGenerator {
                             astChanged = true;
                           }
                         }
+                        // intl.formatMessage({ id: formik.values.message }) || formik.handleChange || { shrink: true } || randomtext
+                        else if (inputProp.value !== prop.initializer.expression.getText()) {
+                            const newAst = createAst(inputProp.value)
```

Comment:
Think about simplification - something like: `updateJsxAttribute(prop, prop.name, newAst)`
For example:
* `prop.value` contains https://ts-ast-viewer.com/#code/EbB0GNKA
* then just check presence of `ExpressionStatement` https://ts-ast-viewer.com/#code/DwYQ9gtgBAhnC8BvARsgdAYywXyA and copy it's children to as new jsx attribute expression

Reply:
why is this code so complicated (it should not be)?